tests/test_trello_pagination.py

```diff
@@ -206,6 +212,12 @@ def test_run(self):
 
         for stream in self.testable_streams():
             with self.subTest(stream=stream):
+        
+                # Page limit for card stream(50) is configured in config above
+                self.API_LIMIT = 50 if stream == "cards" else 1000
```

Comment:
This is set on line 132 already.

Reply:
Yes, but both are under different for loop and we are updating API_LIMIT on for loop iteration so need to set it again.

Reply:
A more consistent implementation would be adding API LIMIT as a piece of expected metadata in base.

Reply:
Yes, that's true but tests written in this tap are not in that format. Every test has its own methods for retrieving expected things and no expected metadata functions and also the pagination test is only for actions and card streams.